pkg/asset/installconfig/aws/validation_test.go

```diff
@@ -168,6 +185,41 @@ func TestValidate(t *testing.T) {
 		}(),
 		availZones:     validAvailZones(),
 		privateSubnets: validPrivateSubnets(),
+	}, {
+		name: "valid instance types",
+		installConfig: func() *types.InstallConfig {
+			c := validInstallConfig()
+			c.Platform.AWS = &aws.Platform{Region: "us-east-1"}
```

Comment:
We already set the region in validInstallConfig to this value https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/validation_test.go#L28 so I don;t think we need to set it again..

Reply:
I either had to set this to override the entire aws section from validInstallConfig or set set `c.Platform.AWS.Subnets = nil` because it would fail on the predefined subnets in validInstallConfig.  Should I be doing it a different way?